index.html

```diff
@@ -57,5 +58,204 @@ <h3 class="contact-title">let's connect</h3>
+							<li>2015</li>
+							<li class="li"></li>
+							<li class="li"></li>
+						</ul>
+					</div>
+					<p>
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required.
+					</p>
+
+					<ul class="tech-used">
+						<li class="tech">html</li>
+						<li class="tech">css</li>
+						<li class="tech">javaScript</li>
+					</ul>
+				</div>
+				<a href="#" class="btn">See Projects</a>
+			</div>
+
+			<div class="single-card">
+				<img
+					src="/images/SnapshootPortfolio-4.png"
+					alt="Portfolio 1 image"
+					class="p-img"
+				/>
+				<div class="about-portfolio">
+					<h1>Multi-Post Stories</h1>
+					<div class="info-portfolio">
+						<ul class="list">
+							<li class="li">canopy</li>
+							<li>Back End Dev</li>
+							<li>2015</li>
+							<li class="li"></li>
+							<li class="li"></li>
+						</ul>
+					</div>
+					<p>
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required.
+					</p>
+
+					<ul class="tech-used">
+						<li class="tech">html</li>
+						<li class="tech">css</li>
+						<li class="tech">javaScript</li>
+					</ul>
+				</div>
+				<a href="#" class="btn">See Projects</a>
+			</div>
+		</section>
+
+		<!-- End of work section -->
+
+		<!-- About me section -->
+		<section class="about-myself">
+			<div class="about-text">
+				<h2>
+					About <br />
+					Myself
+				</h2>
+				<p>
+					Hello i'm a software developer! I can help you build a product,
+					feature or website. Look through some of my work and experience! If
+					you like whatyou see and have a project you need coded, don't hesitate
+					to contact me
+				</p>
+			</div>
+			<div class="contact connect">
+				<h3 class="contact-title">let's connect</h3>
+
+				<ul class="icons">
+					<li>
+						<a href="#" class="social-icon"><i class="fab fa-twitter"></i></a>
+					</li>
+					<li>
+						<a href="#" class="social-icon"><i class="fab fa-linkedin"></i></a>
+					</li>
+					<li>
+						<a href="#" class="social-icon"><i class="fab fa-medium"></i></a>
+					</li>
+					<li>
+						<a href="#" class="social-icon"><i class="fab fa-github"></i></a>
+					</li>
+				</ul>
+			</div>
+			<a href="#" class="btn">Get my resume</a>
+			<ul class="my-skills">
+				<li class="languages skill-section">
+					<div class="skill-title">
+						<h3>Languages</h3>
+						<span class="drop-icon"><i class="fas fa-angle-down"></i> </span>
+					</div>
+					<div class="lang">
+						<img src="./images/Ellipse-js.png" alt="javaScript logo" class="img-logo"/>
+							<span class="lang-name">JavaScript</span>
+						
+					</div>
+					<div class="lang">
+						<img src="./images/Ellipse-HTML.png" alt="javaScript logo" class="img-logo"/>
+							<span class="lang-name">HTML</span>
+						
+					</div>
+					<div class="lang">
+						<img src="./images/Ellipse-css.png" alt="javaScript logo" class="img-logo" />
+							<span class="lang-name">CSS</span>
+					</div>
```

Comment:
- [ ] Kindly add the list of technologies in a `ul` tag with each technology being a `li` tag. This will improve readability, especially for people using screen readers.

Reply:
Sorry, i already tried that but i was getting linter error while using <li> within another <li> or while using <ul> within another <ul>. Hence I resorted to using div.

Reply:
Sorry, I tried but was getting linter error saying can't use li within another li or ul within another ul.
Plus the requirements was to implement as following: 
"Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items."
which I have clearly implemented.